### DISCUSSION TITLE
Waiting for a reply from the user ?

### DISCUSSION THREAD

**Discussion by atakanuludag** [telegraf/telegraf#1551] [Q&A]
Hi, I am using Telegraf v4.
I want to wait for a response from the user?

This code doesn't work because I can't use await;

`bot.on('text', (ctx) => {
    console.log(ctx.message.text);
});`

**Comment by fl3xice** [comment]
```js
bot.on('text', async (ctx) => { console.log(ctx.message.text); });
```
try this :\

**Reply by humblecoder** [reply]
How do you actually provide a `reply to` prompt to the user?  Something to at least "fudge" the idea of a direct response to a question.

I've seen bots post a question, then provide a prompt in reply to that question, but anything I try seems to simply post a reply to the referenced question rather than leave an open prompt for a reply.  🤔

**Reply by MKRhere** [reply]
@humblecoder When sending a message, you must set it to be `force_reply`:

```TS
ctx.sendMessage(
	"Here's a question for you!",
	{
		reply_markup: {
			force_reply: true,
			input_field_placeholder: "Reply with your answer",
		},
	},
);
```

Kindly avoid necroposting on questions not directly related to yours (this is the 3rd time). Create a new issue or discussion with your new question.

**Reply by humblecoder** [reply]
Thanks.  I appreciate the speedy response.  

> Kindly avoid necroposting on questions not directly related to yours (this is the 3rd time). Create a new issue or discussion with your new question

Yes, with regard to "necroposting", my previous posts were addendums to topics for which there were already several "dead" posts.  I didn't feel that adding another post to "die" would be any more useful.

With regard to this specific item, since it was tangentially related to my question, and never "closed" or marked "answered" (and was less than a year old), I didn't consider it dead.

Thanks again for your answer.

**Reply by dixpie** [reply]
How can i get answer ?
